Packs/AWS-S3/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "AWS - S3",
     "description": "Amazon Web Services Simple Storage Service (S3)",
     "support": "xsoar",
-    "currentVersion": "1.1.4",
+    "currentVersion": "1.1.5",
```

I suggest a minor update, as it contains new functionality.
```suggestion
    "currentVersion": "1.2.0",
```

---

Done :)